src/Castle.Windsor/Castle.Windsor.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
 	<PropertyGroup>
-		<TargetFrameworks>net45;netstandard1.6</TargetFrameworks>
+    <TargetFrameworks>net45;netstandard2.0;net5.0</TargetFrameworks>
```

Comment:
In summary, we are dropping .NET Standard 1.6 and adding .NET Standard 2.0 and .NET 5.0, correct?

.NET Standard 2.0 for .NET Framework 4.6.1+, .NET Core 2.x and .NET Core 3.x. .NET Core 1.x is unsupported now anyway.

Reply:
I read this on a MS page. I.e. that this was the recommendation (of cause right now, I cannot find again)
net45 to net46 = net45
net461 to net48 and netappcore2.0 to netapp3.1 = netstandard2.0
net5.0 = net 5.0

this will mean ofcause that netapp1.x will not be able to update.

Reply:
https://docs.microsoft.com/en-us/dotnet/standard/net-standard

What value does adding a `net5.0` target add? .NET 5 supports .NET Standard 2.0 so it should be unnecessary right? And is just increasing build and test time, and package size?

Reply:
I read the MS recommends net5.0 targetting for packaging. 
Don't you think, making a net5.0 specific build, will build the packages with the specific improvements from the net5.0 sdk and increase e.g. increase runtime performance for net5 applications (or am I just dreaming)?

I can delete if you insist :-)

Reply:
I don't insist on anything, I'd just like to understand the guidance from the .NET team as I probably know less about .NET 5 than you do. I do recall reading somewhere that .NET Standard is now seen as "mature" (i.e. pretty much end of life) and that the `net5.0` target is the future, but wasn't sure whether that was the official position or .NET Standard libraries are still good. I can't see there being any difference in performance over which TFM you choose as it is just an API surface and still the same runtime.

[The future of .NET Standard](https://devblogs.microsoft.com/dotnet/the-future-of-net-standard/) blog post on the .NET Blog talks about the difference. My understanding is that offering a `net5.0` TFM will make no difference since we will also have a `netstandard2.0` one (.NET 5 supports all .NET Standard versions), until we want to use .NET 5 features which this will enable.

I'm happy for this to include the `net5.0` if you think you'll want to use features in .NET 5, but those will have to be optional to keep support for older TFMs.

If anyone else reading along has more info please jump in.